api-specifications/properties.json

```diff
@@ -0,0 +1,246 @@
+{
+  "scriptappy": "1.0.0",
+  "info": {
+    "name": "@nebula.js/sn-org-chart:properties",
+    "description": "Org chart generic object definition",
+    "version": "0.15.0",
+    "license": "MIT",
+    "stability": "experimental",
+    "x-qlik-visibility": "public"
+  },
+  "entries": {
+    "properties": {
+      "extends": [
+        {
+          "type": "qae.GenericObjectProperties"
+        }
+      ],
+      "kind": "object",
+      "entries": {
+        "footnote": {
+          "description": "Visualization footnote",
+          "defaultValue": "",
+          "type": "string"
+        },
+        "navigationMode": {
+          "description": "How the org chart is navigated",
+          "defaultValue": "free",
+          "kind": "union",
+          "items": [
+            {
+              "kind": "literal",
+              "value": "'regular'"
+            },
+            {
+              "kind": "literal",
+              "value": "'scroll'"
+            },
+            {
+              "kind": "literal",
+              "value": "'free'"
+            }
```

Comment:
These are not the correct options right? I think there is free and expandAll mode. We have talked about different modes, but never implemented any